grafana-backup/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -l
+# shellcheck disable=SC1090
+
+# This script copies Grafana dashboard jsons, tar them and put into HDFS and EOS folder.
+# Ref for alerting: https://github.com/dmwm/CMSSpark/blob/master/bin/cron4aggregation
+
+##H Usage: run.sh KERBEROS_KEYTAB TOKEN_LOCATION HDFS_PATH FILESYSTEM_PATH
+##H Example:
+##H        run.sh keytab keys/token.json /cms/backups/grafana/ /eos/cms/store/group/offcomp_monit/
+
+# help definition
+if [ "$1" == "-h" ] || [ "$1" == "-help" ] || [ "$1" == "--help" ] || [ "$1" == "help" ] || [ "$1" == "" ]; then
+    grep "^##H" <"$0" | sed -e "s,##H,,g"
+    exit 1
+fi
+
+# Get utils functions
+. "${script_dir}"/../scripts/utils.sh
```

Comment:
I saw that on other occasions utils.sh is copied to the same folder, but is it possible to do it this way?